src/Cryptol/Utils/Ident.hs

```diff
@@ -58,7 +58,7 @@ module Cryptol.Utils.Ident
   , modParamIdent
```

Comment:
Is this `modParamIdent` function actually used anywhere? I ask since it creates an `Ident` whose text begins with `"module parameter"`, which seems like the sort of verbose identifier that this patch is working to fix.

Reply:
It appears to be unused in the Cryptol code base, so I'll delete it.